packages/paste-website/stories/inline-code.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/core/anchor';
+import {InlineCode} from '../src/components/Typography';
+
+export const RegularInline = (): React.ReactNode => <InlineCode>string</InlineCode>;
+
+export const AnchorInline = (): React.ReactNode => (
+  <InlineCode>
+    <Anchor href="#">string</Anchor>
+  </InlineCode>
+);
+
+export const ExternalAnchorInline = (): React.ReactNode => (
+  <InlineCode>
+    <Anchor href="#" showExternal>
+      string
+    </Anchor>
+  </InlineCode>
+);
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Website/Inline-Code',
+  component: RegularInline,
+};
```

Comment:
It miiiight be worth adding an example that has two inline code snippets stacked on top of each other to see what it looks like without having to search through the docs page, and maybe one for inside of a callout (though I guess that would be most relevant for when the callout PR is merged in...)

Reply:
truuuue